dlp/dlp_inspect_image_file.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START dlp_inspect_image_file]
+require "google/cloud/dlp"
+
+def inspect_image_file (
+    project_id = "YOUR_PROJECT_ID",
+    filepath   = "path/to/image.png"
```

Comment:
Specifically, don't use default arguments as documentation. Use them for true defaults. If you need to provide sample data for tests, do so in the line at the bottom where you invoke this method, or better, from the test runner.